plugins/vli/fu-vli-pd-device.c

```diff
@@ -892,6 +904,9 @@ fu_vli_pd_device_init(FuVliPdDevice *self)
 	fu_device_register_private_flag(FU_DEVICE(self),
 					FU_VLI_PD_DEVICE_FLAG_HAS_I2C_PS186,
 					"has-i2c-ps186");
+	fu_device_register_private_flag(FU_DEVICE(self),
+					FU_VLI_PD_DEVICE_FLAG_SKIPS_ROM_MODE,
+					"skips-rom");
```

Comment:
i think you either want `skips-rom-mode` or `FU_VLI_PD_DEVICE_FLAG_SKIPS_ROM` -- but I have no problem conceptually with the patch.